notebooks/Notebook1.md

```diff
@@ -9,6 +9,29 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+
+
+
+I think I made a lot of progress this week on starting the project and have some ideas of where to start!
+	The feedback received in bullet points is as follows:
+- It’s best to use a library if it exists for the host language being used.
+- Both Python and Java have libraries that will be useful
+- There are still questions about how the schedule will be visualized, not only in what file type and format but how the user will see the calendar (daily, weekly, monthly?”
+- Sounds very extreme, but can be exactly what some niche users are looking for. 
+- Should focus on efficiency instead of features.
+- Choices can be made to skew towards language design vs systems design.
+- PyQt in Python is a free and simple way to make a GUI that can be used for a frontend.
+- The frontend doesn’t have to look ridiculously professional.
+- Plan is a nice balance of knowing what milestones to complete, while also having some flexibility if you need it. 
+- Between Python, Scala, and Rust, Python will most likely give the best balance of language design and systems design.
+
+Some of this feedback has already been considered and will be used when building the project in week 2. For instance, Python will be used instead of Scala as the host language. This ensures that libraries exist that will offset much of the systems design work and balance the project towards language design. This also allows in the future for the development of a frontend, which will most likely use PyQt or Tkinter, which was recommended in feedback. I had originally considered a desktop app using a mix of Javascript/Rust purely because that’s what I already have experience with, but from feedback I decided it wasn’t worth complicating every aspect of the project just to have a slightly prettier frontend. 99%, if not 100%, of what I would want to do in Javascript/Rust would also be possible using Python.  
+	Something I hadn’t considered in detail is how the calendar/schedule will be visualized. I had assumed that it could look something similar to Google Calendar/Apple calendar, but people use calendars in different ways and one might want to view their schedule for the day or for the week/month. For this reason, the output file to visualize the schedule will be a .csv file.
+
```

Comment:
I'm curious: Can you say more about why you are choosing a .csv file for visualization? Is this to provide the end-user with more flexibility? Or are you imagining that the .csv file is organized in a particular way such that when it is opened in Excel (or Google Sheets, or ...), it will encode a particular way of visualizing the calendar?